model.py

```diff
@@ -6847,18 +6847,24 @@ def circulation_changelog(self, old_licenses_owned, old_licenses_available,
         logging.info or a similar method
         """
         edition = self.presentation_edition
-        message = 'CHANGED '
+        message = u'CHANGED '
         args = []
+        if self.identifier:
+            identifier_template = '%s/%s'
+            identifier_args = [self.identifier.type, self.identifier.identifier]
+        else:
+            identifier_template = '%s'
+            identifier_args = [self.identifier]
         if edition:
-            message += '%s "%s" %s (%s)'
+            message += u'%s "%s" %s (' + identifier_template + ')'
             args.extend([edition.medium, 
                          edition.title or "[NO TITLE]",
-                         edition.author or "[NO AUTHOR]",
-                         self.identifier]
+                         edition.author or "[NO AUTHOR]"]
                     )
+            args.extend(identifier_args)
         else:
-            message += '%s'
-            args.append(self.identifier)
+            message += identifier_template
+            args.extend(identifier_args)
```

Comment:
This change is to fix an annoying issue where a book with a non-ASCII character in its title can't have its circulation information logged. The title itself is all right, but the representation of the book's Identifier _also_ includes the title, and turning the representation of the book's identifier into a '%s' causes a UnicodeDecodeError.

Since it's not necessary to include the title twice, this change simplifies things and eliminates the problem by only logging the identifier and identifier type.